src/main/java/org/opentripplanner/middleware/auth/Auth0Users.java

```diff
@@ -269,6 +266,15 @@ public static String getAuth0Token(String username, String password) throws Json
             LOG.error("Cannot obtain Auth0 token for user {}. response: {} - {}", username, response.statusCode(), response.body());
             return null;
         }
-        return getSingleNodeValueFromJSON("access_token", response.body());
+        return JsonUtils.getPOJOFromJSON(response.body(), TokenHolder.class);
+    }
+
+    /**
+     * Extract from a complete Auth0 token just the access token. If the token is not available, return null instead.
+     */
+    public static String getAuth0Token(String username, String password) {
```

Comment:
I suggest renaming this method to `getAuth0AccessToken`.

Reply:
Updated.